cspell.config.json

```diff
@@ -48,6 +48,7 @@
     "gdev",
     "lokiexplore",
     "viewports",
+    "Descendents",
```

Comment:
Shouldn't this be **Descendants**, not [Descendents](https://en.wikipedia.org/wiki/Descendents)?

Reply:
Yes, but it's a common misspelling and I wanted to be [consistent](https://github.com/grafana/scenes/blob/33d096e023645c65fad2aa8ad2ba63c5968a5207/packages/scenes/src/core/sceneGraph/sceneGraph.ts#L257) :)

Reply:
![](https://media2.giphy.com/media/v1.Y2lkPTc5MGI3NjExdzZ0NjNzaXB2bXF1eW15N25yc3dqdHBqa2ZtMzBld2xnajN2OWgwMCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/uHlGWE1DyjWFkXFRFG/giphy.gif)